greedy_algorithms/dijkstra.cpp

```diff
@@ -32,7 +41,14 @@ class Graph {
         this->edges[src][dst] = weight;
     }
 };
-// Utility function to find minimum distance vertex in mdist
+/**
+ * Function to find minimum distance vertex in mdist.
+ * @param mdist[] array of minimum distances to vertices
+ * @param vset[] array of vertices, where vset[i] is a bool that indicates
+ * whether the vertex i is included in the shortest path tree
+ * @param int number of vertices
+ * @return minimum distance vertex
+ */
 int minDistance(int mdist[], bool vset[], int V) {
```

Comment:
```suggestion
int minDistance(const int *mdist, const bool *vset, int V) {
```
Arrays should be passed as pointers. Since the arrays are not being modified, pass them as `const pointers` to prevent data corruption